plugins/node/instrumentation-fs/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@opentelemetry/instrumentation-fs",
   "version": "0.12.0",
-  "description": "OpenTelemetry automatic instrumentation package for fs",
+  "description": "OpenTelemetry instrumentation for `fs` file system interactions NodeJS core module.",
```

Comment:
nit: I'd be inclined to use `Node.js` rather than `NodeJS`.

Reply:
done 👍